nautobot/docs/rest-api/overview.md

```diff
@@ -785,3 +785,478 @@ http://nautobot/api/dcim/sites/ \
 
 !!! note
     The bulk deletion of objects is an all-or-none operation, meaning that if Nautobot fails to delete any of the specified objects (e.g. due a dependency by a related object), the entire operation will be aborted and none of the objects will be deleted.
+
+### Depth Query Parameter
+
++++ 2.0.0
+
+The the built-in DRF `?depth` query parameter is introduced in Nautobot 2.0 to replace the `?brief` parameter. It enables [nested serialization](https://www.django-rest-framework.org/api-guide/serializers/#specifying-nested-serialization) functionality and offers a more dynamic and comprehensive browsable API.
+This parameter is an positive integer value that can range from 0 to 10. In most use cases, you will only need a maximum `depth` of 2 to get all the information you need.
+
+!!! important
+    The `?depth` query parameter should only be used for `GET` operations in the API. It should not be used in `POST`, `PATCH` and `DELETE` requests. For these requests, only `?depth=0` should be used.
```

Comment:
Second-guessing myself on this point - since a PUT/POST/PATCH returns the equivalent of a GET response, **should** it be possible to specify the `depth` that you want that response to provide?